src/doc.d

```diff
@@ -1122,7 +1126,7 @@ extern (C++) void toDocBuffer(Dsymbol s, OutBuffer* buf, Scope* sc)
             // emit constraints if declaration is a templated declaration
             if (td && td.constraint)
             {
-                buf.writestring(" if (");
+                buf.writestring("$(DDOC_CONSTRAINT ");
```

Comment:
In DDoc/compiler:
```
DDOC_CONSTRAINT=$0
```
In Phobos configuration:
```
DDOC_CONSTRAINT=<span class="constraint">$0</span>
```
This way, other users of DDoc won't have surprise changes to their output. Especially considering HTML isn't the only possible target language; for non-HTML targets it would just outright break!